get-thin-edge_io.sh

```diff
@@ -81,14 +81,14 @@ if [ -t 1 ]; then
     fi
 fi
 
-echo "${BLUE}Thank you for trying thin-edge.io! ${COLORRESET}\n"
+echo -e "${BLUE}Thank you for trying thin-edge.io! ${COLORRESET}\n"
```

Comment:
In order to be `sh` compatible.

```suggestion
echo "${BLUE}Thank you for trying thin-edge.io! ${COLORRESET}"
echo
```

Reply:
Colors are also escape sequences.

Reply:
But do we really care about colors for a quick install script? Portability matters more.

Reply:
I cannot decide that, but if you think I can remove the colors, I'm happy to rewrite this portion of the patchset to make sure we use `/bin/sh` and only features that are there according to `shellcheck` (thus removing colors)! :+1: